docs/selecting-a-data-source.md

```diff
@@ -0,0 +1,37 @@
+# Selecting a Data Source
+
+The Carbon Aware SDK includes access to various data sources of carbon aware data, including WattTime, ElectricityMaps, and a custom JSON source. These matrices are an attempt to track what features of the Carbon Aware SDK are enabled for which data sources.
+
+## Contents
+
+- [Type of Data Sources and Configuration](#type-of-data-sources-and-configuration)
+- [Data Source Methods Available](#data-source-methods-available)
+- [Location Coverage](#location-coverage)
+
+## Type of Data Sources and Configuration
+
+In the CarbonAware SDK configuration, you can set what data source to use as the `EmissionsDataSource` and the `ForecastDataSource`. There are also certain configuration fields that must be set in order to access the raw data.
+| Type | WattTime  | ElectricityMaps | JSON |
+|--------------------------|-----------|-----------------|------|
+| Is Emissions DataSource | &#9989; | &#9989; | &#9989; |
+| Is Forecast DataSource | &#9989;  | &#9989; | &#10060; |
+| Makes HTTP(s) call | &#9989;  | &#9989; | &#10060; |
+| Can Use Custom Data | &#10060;  | &#10060; | &#9989; |
+| Supports Trial + Full Account | &#9989;  | &#9989; (*[different config required](./configuration.md#electricitymaps-configuration)) | N/A |
+
+## Data Source Methods Available
+
+Not all data sources support all the routes provided in the interfaces (`IEmissionsDataSource`/`IForecastDataSource`).
+
+| Methods | WattTime  | ElectricityMaps | JSON | CLI Usage | Web Api Usage | SDK Usage
+|--------------|:-----------:|:-----------------:|:------:|:-:|:-:|:-:|
+| GetCarbonIntensityAsync | &#9989;  | &#9989; | &#9989; |`emissions`|`emissions/bylocation` or `emissions/bylocations` or `emissions/bylocations/best` or `emissions/average-carbon-intensity` or `average-carbon-intensity/batch`|`GetEmissionsDataAsync(...)` or `GetBestEmissionsDataAsync(...)` or `GetAverageCarbonIntensityDataAsync(...)`|
```

Comment:
The forced wrapping on the hyphenated paths makes this table a bit tricky to read (IMO)

Consider swapping out the hyphen character for the [non-breaking hyphen](https://www.fileformat.info/info/unicode/char/2011/index.htm) `&#8209;` 

`<code>emissions/average&#8209;carbon&#8209;intensity</code>`